modules/galaxy/tasks/milkywaypointsconversiontask.cpp

```diff
@@ -22,35 +22,38 @@
  * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
  ****************************************************************************************/
 
-#include <apps/DataConverter/milkywaypointsconversiontask.h>
+#include <modules/galaxy/tasks/milkywaypointsconversiontask.h>
 #include <modules/volume/textureslicevolumereader.h>
 #include <modules/volume/rawvolumewriter.h>
 #include <modules/volume/volumesampler.h>
 #include <fstream>
 #include <iostream>
 
 namespace openspace {
-namespace dataconverter {
-    
 
-
-MilkyWayPointsConversionTask::MilkyWayPointsConversionTask(
+/*MilkywayPointsConversionTask::MilkywayPointsConversionTask(
     const std::string& inFilename,
     const std::string& outFilename)
     : _inFilename(inFilename)
-    , _outFilename(outFilename) {}
+    , _outFilename(outFilename) {}*/
+
+MilkywayPointsConversionTask::MilkywayPointsConversionTask(const ghoul::Dictionary & dictionary) {}
 
+MilkywayPointsConversionTask::~MilkywayPointsConversionTask() {}
 
-void MilkyWayPointsConversionTask::perform(const std::function<void(float)>& onProgress) {
+std::string MilkywayPointsConversionTask::description()
+{
+    return std::string();
+}
+
+void MilkywayPointsConversionTask::perform(const Task::ProgressCallback & progressCallback) {
     std::ifstream in(_inFilename, std::ios::in);
     std::ofstream out(_outFilename, std::ios::out | std::ios::binary);
-      
+
     std::string format;
     int64_t nPoints;
     in >> format >> nPoints;
 
-
- 
     size_t nFloats = nPoints * 7;
 
     float* pointData = new float[nFloats];
```

Comment:
You are leaking the pointData,  better to use a `std::vector<float> pointData(nFloats)`